index.html

```diff
@@ -0,0 +1,114 @@
+        </li>
+        <li id="experience-2" class="experience">
+          <div class="experience-image-2"></div>
+          <h4 class="experience-main">Props</h4>
+          <p class="experience-info">Explore the collection of beautifully crafted iconic props</p>
+        </li>
+        <li id="experience-3" class="experience">
+          <div class="experience-image-3"></div>
+          <h4 class="experience-main">Costumes</h4>
+          <p class="experience-info">See up close the intricate detail hidden in every costume</p>
+        </li>
+        <li id="experience-4" class="experience">
+          <div class="experience-image-4"></div>
+          <h4 class="experience-main">Effects</h4>
+          <p class="experience-info">Discover how the special effects teams made Harry and his friends fly</p>
+        </li>
+        <li id="experience-5" class="experience">
+          <div class="experience-image-5"></div>
+          <h4 class="experience-main">Creatures</h4>
+          <p class="experience-info">Learn how magical creatures were brought to life on screen</p>
+        </li>
+        <li id="experience-6" class="experience">
+          <div class="experience-image-6"></div>
+          <h4 class="experience-main">Art</h4>
+          <p class="experience-info">From technical drawings to scale models, learn how sets were created</p>
+        </li>
+      </ul>
+      <button id="find-out-btn"><a href="" id="find-out-btn-link">Find out more</a></button>
+      <a href="" id="find-out-btn-link-lg">Find out more</a>
+    </section>
+    <section id="featured" class="featured-section">
+      <h3 class="featured-heading">What's On</h3>
+      <div id="featured-heading-under"></div>
+      <ul class="featured"></ul>
+    </section>
+    <section id="awards" class="awards-section">
+      <h3 class="awards-heading">Award-Winning Tour</h3>
+      <div id="awards-heading-under"></div>
+      <ul class="awards-list">
+        <li class="award"><img src="https://s32508.pcdn.co/wp-content/uploads/2022/03/PngItem_6092654-1-295x300.png" alt="award image" class="award-img"/></li>
+        <li class="award"><img src="https://s32508.pcdn.co/wp-content/uploads/2022/03/TC_L-300x300.png" alt="award image" class="award-img"/></li>
+        <li class="award"><img src="https://s32508.pcdn.co/wp-content/uploads/2019/06/gta_2019_1-300x300.jpg" alt="award image" class="award-img"/></li>
+        <li class="award"><img src="https://s32508.pcdn.co/wp-content/uploads/2019/12/award-dowtk-national-winners-big-kids-300x300.png" alt="award image" class="award-img"/></li>
+        <li class="award"><img src="https://s32508.pcdn.co/wp-content/uploads/2022/03/2019-Tourism-Awards-Logo-Herts-Winner-2019-AW-300x300.png" alt="award image" class="award-img"/></li>
+      </ul>
+    </section>
+  </main>
+  <footer>
+    <div class="footer-logo">
+      <img src="./media/Warner-Bros-Logo-1.png" alt="footer-logo-image" class="logo-img-footer">
```

Comment:
- [ ] The same applies to the `footer` section. Kindly adjust the background color to match the original design

![image](https://user-images.githubusercontent.com/98453979/204054321-76e62a91-fd8d-4076-bdfa-6d86d65f3257.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/204054366-75eadfe3-24ee-4015-97bd-98c54381bb4f.png)